public/js/clients/chrome.js

```diff
@@ -42,12 +45,19 @@ function presentTabs(tabs) {
     });
 }
 
-function chromeTabs(callback) {
+function connectClient() {
+  if (!isEnabled("chrome.debug")) {
+    return Promise.resolve([]);
+  }
+
+  const deferred = defer();
   fetch("/chrome-tabs").then(res => {
     res.json().then((body) => {
-      callback(presentTabs(body));
+      deferred.resolve(formatTabs(body));
```

Comment:
switched from callback pattern to promises so that I could call both clients with Promise.all